web/src/tippyjs.js

```diff
@@ -382,6 +385,74 @@ export function initialize() {
             instance.destroy();
         },
     });
+    delegate("body", {
+        target: ".rendered_markdown .previewable",
+        appendTo: () => document.body,
+        content() {
+            return parse_html(tooltip_loader());
+        },
+        maxWidth: "350px",
+        delay: [300, 20],
+        onShow(instance) {
+            channel.post({
+                url: "/json/previewable",
+                data: {
+                    url: $(instance.reference).attr("href"),
+                },
+                success(data) {
+                    let state;
+                    // If adding more platforms support then make a function to get the data, as platform is key in data we get from backend, Ex - github.
+                    if (data.type === "pulls") {
+                        state = data.state;
+                        const merged = data.pull_request.merged_at !== null;
+                        if (data.draft) {
+                            state += "-draft";
+                        } else if (merged) {
+                            state += "-merged";
+                        }
+                    } else {
+                        state = [data.state, data.state_reason].filter(Boolean).join("-");
+                    }
+                    instance.setContent(
+                        parse_html(
+                            previewable_url_tooltip({
+                                hover_preview_title: data.title,
+                                hover_preview_details: $t(
+                                    {
+                                        defaultMessage:
+                                            "{ owner }/{ repo }#{ issue_number } opened by { user }",
```

Comment:
I think it'd be nicer to construct a `{issue}` variable for the first part, and have this be spelled for translations `"{issue} opened by {user}"`. (We usually don't put extra space around variable names).